From cf5eee1047357b02baaadc992fa2c58c94d38dc2 Mon Sep 17 00:00:00 2001
From: khaiwenn <khai.wen.ng@intel.com>
Date: Thu, 28 Feb 2019 16:38:18 +0800
Subject: [PATCH 1307/1443] Adding delay after initializing OV10635

OV10635 requires a maximum of 10ms holding time
during power on. Set the delay to 10ms to make
sure the sensor is well init before turning it
off.

Change-Id: Ib704da2c610382f8e159156266cdbb3ac31b89e1
Signed-off-by: khaiwenn <khai.wen.ng@intel.com>
---
 drivers/media/i2c/ici/ti964_ici.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/media/i2c/ici/ti964_ici.c b/drivers/media/i2c/ici/ti964_ici.c
index ec91295..dafcf69 100644
--- a/drivers/media/i2c/ici/ti964_ici.c
+++ b/drivers/media/i2c/ici/ti964_ici.c
@@ -426,6 +426,8 @@ static int ti964_registered(struct ici_ext_subdev_register *reg)
 		/* Initialize sensor connected to TI964 */
 		rval  = ti964_init_ext_subdev(info, reg, client,
 						  &sd_register);
+		msleep(10);
+
 		if (rval) {
 			pr_err("%s, Failed to register external subdev\n", __func__);
 			continue;
-- 
2.7.4

